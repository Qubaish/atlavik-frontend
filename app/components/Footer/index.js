import React from 'react';
import { FormattedMessage } from 'react-intl';

import Wrapper from './Wrapper';
import messages from './messages';

function Footer() {
  return (
    <Wrapper>
      <section>
        <FormattedMessage {...messages.licenseMessage} />
      </section>
      <section>
        <FormattedMessage
          {...messages.authorMessage}
          values={{
            author: <a href="https://twitter.com/mxstbr">Qubaish</a>,
          }}
        />
      </section>
    </Wrapper>
  );
}

export default Footer;