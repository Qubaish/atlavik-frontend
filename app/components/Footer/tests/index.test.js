import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Footer from '../index';

describe('<Footer />', () => {
    it('should render a DropDown', () => {
        const container = shallow(
            <Footer />
        );
        expect(shallowToJson(container)).toMatchSnapshot();
    });
});
