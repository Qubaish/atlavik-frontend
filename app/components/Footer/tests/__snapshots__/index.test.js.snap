// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render a DropDown 1`] = `
<Wrapper>
  <section>
    <FormattedMessage
      defaultMessage="This project is licensed under the MIT license."
      id="boilerplate.components.Footer.license.message"
      values={Object {}}
    />
  </section>
  <section>
    <FormattedMessage
      defaultMessage="
      Made with love by {author}.
    "
      id="boilerplate.components.Footer.author.message"
      values={
        Object {
          "author": <a
            href="https://twitter.com/mxstbr"
          >
            Qubaish
          </a>,
        }
      }
    />
  </section>
</Wrapper>
`;
